### ISSUE TITLE
Specify fields for  new record type Banns

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#1427]
Priority (0 3 2 3) 8
Rather than adding dates of banns to a marriage document, it was decided to create a new record type.
The minimum fields are the 3 dates when banns were read and the names of the persons. 
Are there other fields?
blocks #1453

**Comment by SteveBiggs** [comment]
Yes, Reg number, condition of bride and groom, parish of bride and groom:

![image](https://user-images.githubusercontent.com/25303607/37856197-0f0831d6-2eeb-11e8-9ab3-0967ce233e21.png)
and would be good to have a Notes field.

**Comment by Captainkirkdawson** [comment]
Should have mentioned that the common fields would be there
County
Place
Church
Register type
Register entry number
Register notes
Transcriber notes
Film No.
Image No.
Image File Name

The suggestions are good TY

**Comment by Sherlock21** [comment]
Do you need to show if the Marriage was actually completed, of if these banns got abandoned or cancelled - as some of mine did in the period where the banns + subsequent marriage are all on the one form.

Or is that spoon feeding too much?

**Comment by edickens** [comment]
And Title of the people plus any of the other new fields we are thinking of for the people.

**Comment by edickens** [comment]
But **Banns** raises an interesting issue.  Researchers will want them linked to the marriage.  But this is not transcribing.  So when we get to Researcher's enhancements, will we let them establish the link, rather like the other genealogy websites which are building family trees.  And then do we link to FreeBMD and FreeCEN?

**Comment by edickens** [comment]
Another thought.  Quite a few Banns entries are crossed out or do not have all three entries.  I suspect this is because the marriage did not go ahead.  This is still useful information as it puts someone at a place and time.  The transcriber should do these entries and make a note, but is it important enough to have a field so that analysis of incomplete banns can be analysed?

**Comment by SteveBiggs** [comment]
If the Banns & Marriage are on one form, the Marriage itself should have been transcribed separately, so it's not so important to record the Banns as a separate type. The new Banns record type should be for Banns only records like the example I showed a few posts up as this may be the only record of the couple being together.

If the Banns were abandoned or crossed out, that should be recorded in the Notes (as we do with other types). Adding fields for this seems to me to be getting too cumbersome, but I'm open to others' views.

**Comment by Sherlock21** [comment]
We don't record BANNS at present - even if they re on a common form with the subsequent Marriage. So as I say above, these BANNS will NOT have been Transcribed.
Yet Steve  says: the new BANNS record type should be for BANNS ONLY Records...

SO where do the BANNS that are on a common form with Marriages go?

**Comment by SteveBiggs** [comment]
The motivation for recording Banns is to capture people who may not already be recorded. If they are already recorded on a Marriage record, there is little value in re-transcribing that Marriage into a Banns record. At least if we do, it should be done after all the Banns only records are transcribed.

**Comment by Sherlock21** [comment]
I therefore fail to understand what the exercise is.

Are you really suggesting they BANNS and MARRIAGES are to be on the one and same form. - i.e. the same batch & data?

**Comment by SteveBiggs** [comment]
I think we are talking at cross purposes, Eric.

Some forms did indeed have Marriages and Banns together and when these Marriages were transcribed, some transcribers recorded the Banns dates in the Notes.  But whether they did or not, the couple have been recorded in the Marriage transcription.

What I'm saying is that the most important Banns to transcribe are the ones that are NOT on the same form as the as the Marriage because we don't know whether or not there was an associated Marriage, so the couple may not have been recorded elsewhere.

The Banns that are on the same form should be transcribed but as a lower priority.

**Comment by SteveBiggs** [comment]
I have created a spreadsheet of suggested fields for Banns here:

https://docs.google.com/spreadsheets/d/1iW5n7lK9iOZgg73VeK_o-421YsZfYp0dYmb9dcG_tiU/edit?usp=sharing

Please review and amend as you see fit.

**Comment by AlOneill** [comment]
What does "If different to code" (in the Type column) mean/imply?

**Comment by SteveBiggs** [comment]
I don't know. It was there from something I copied. I suggest you ignore it. Have deleted.

**Comment by SteveBiggs** [comment]
I'm keen to get started on implementing the Banns record type so if no one has any other comments are we good with this?

**Comment by richpomfret** [comment]
In that case we'll move forward with this and the next story. Steve to send out link for final feedback/request.

**Comment by SteveBiggs** [comment]
Message sent to all coordinators and transcribers asking them to review proposed fields.

**Comment by SteveBiggs** [comment]
Have had a lot of responses and updated the fields accordingly. Please can everyone have another look to see if we have everything we need? Assume we need to wait for Kirk to return before we can progress anyway?
